index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <b>Hello Microverse</b>
```

Comment:
Please ensure you use `h1` tag for the `Hello Microverse` according to the requirement.

Reply:
@ibrahim777764  Kindly re-submit for re-review once you are done with all the required changes.